EdFi.Buzz.UI.Angular/src/app/Components/StudentCardReact/studentCard.tsx

```diff
@@ -2,7 +2,158 @@ import { Student } from 'src/app/Models';
 import { useState } from 'react';
 import * as React from 'react';
 
-import './studentCard.css';
+import './StudentCard.css';
```

Comment:
```suggestion
import './studentCard.css';
```
If case are different I get 
WARNING in ./src/app/Components/StudentCardReact/StudentCard.css 
There are multiple modules with names that only differ in casing.